pipeline/merger/src/test/scala/weco/pipeline/merger/MergerIntegrationTest.scala

```diff
@@ -299,89 +373,66 @@ class MergerScenarioTest
-      outcome.getMerged(workForMets) should beRedirectedTo(workForEbib)
+      context.getMerged(workForMets) should beRedirectedTo(workForEbib)
 
       And("the Sierra e-bib gets the items from the METS work")
-      outcome.getMerged(workForEbib).data.items shouldBe workForMets.data.items
+      context.getMerged(workForEbib).data.items shouldBe workForMets.data.items
 
       And("the Sierra physical work is unaffected")
-      outcome.getMerged(workWithPhysicalVideoFormats) shouldBe workWithPhysicalVideoFormats
+      context
+        .getMerged(workWithPhysicalVideoFormats)
+        .data shouldBe workWithPhysicalVideoFormats.data
+      context.getMerged(workWithPhysicalVideoFormats).state should beSimilarTo(
+        workWithPhysicalVideoFormats.state)
     }
+  }
 
-    Scenario("A Tei and a Sierra digital and a sierra physical work are merged") {
+  Scenario("A Tei and a Sierra digital and a sierra physical work are merged") {
+    withContext { implicit context =>
       Given("a Tei, a Sierra physical record and a Sierra digital record")
       val (digitalSierra, physicalSierra) = sierraIdentifiedWorkPair()
-      val teiWork = teiIdentifiedWork().title("A tei work")
+      val teiWork = teiIdentifiedWork()
+        .mergeCandidates(
+          List(createTeiBnumberMergeCandidateFor(physicalSierra)))
 
-      When("the works are merged")
-      val sierraWorks = List(digitalSierra, physicalSierra)
-      val works = sierraWorks :+ teiWork
-      val outcome = merger.merge(works)
+      When("the works are processed by the matcher/merger")
+      processWorks(digitalSierra, physicalSierra, teiWork)
 
       Then("the Sierra works are redirected to the tei")
-      outcome.getMerged(digitalSierra) should beRedirectedTo(teiWork)
-      outcome.getMerged(physicalSierra) should beRedirectedTo(teiWork)
+      context.getMerged(digitalSierra) should beRedirectedTo(teiWork)
+      context.getMerged(physicalSierra) should beRedirectedTo(teiWork)
 
       And("the tei work has the Sierra works' items")
-      outcome
+      context
         .getMerged(teiWork)
         .data
         .items should contain allElementsOf digitalSierra.data.items
-      outcome
+      context
         .getMerged(teiWork)
         .data
         .items should contain allElementsOf physicalSierra.data.items
 
       And("the tei work has the Sierra works' identifiers")
-      outcome
+      context
         .getMerged(teiWork)
         .data
         .otherIdentifiers should contain allElementsOf physicalSierra.data.otherIdentifiers
         .filter(_.identifierType == IdentifierType.SierraIdentifier)
-      outcome
+      context
         .getMerged(teiWork)
         .data
         .otherIdentifiers should contain allElementsOf digitalSierra.data.otherIdentifiers
         .filter(_.identifierType == IdentifierType.SierraIdentifier)
     }
+  }
 
-    Scenario("A Tei with internal works and a Calm are merged") {
-      Given("a Tei and a Calm record")
```

Comment:
There's nothing in the TEI transformer to set up this relationship (instead, it goes via Sierra b numbers), so I removed this test.